example/server/pom.xml

```diff
@@ -60,7 +60,7 @@
             </relocation>
             <relocation>
               <pattern>com.google.common.</pattern>
-              <shadedPattern>cz.o2.proxima.internal.shaded.com.google.common.</shadedPattern>
+              <shadedPattern>cz.o2.proxima.kafka.shaded.com.google.common.</shadedPattern>
```

Comment:
This looks a little strange to me, why does `example` shade its dependencies to `kafka` package?

Reply:
I'll check it ounce again tomorrow. You are right, this doesn't make sense and it's weird. Maybe some issue with maven 3.2 - i'll check it again with 3.5

Reply:
Reverted back to original value.